plugins/dune-rules/test/test.t

```diff
@@ -65,41 +49,18 @@ generated.
 When the optional output argument is set, rules will be written in the file and will reflect
```

Comment:
The end of this sentence talks about the now-obsolete rule to generate dune rules, doesn’t it?

Reply:
Yes it is :-)